olive/model/handler/base.py

```diff
@@ -36,12 +38,17 @@ def __init__(
         model_file_format: ModelFileFormat,
         model_path: OLIVE_RESOURCE_ANNOTATIONS = None,
         model_attributes: Optional[Dict[str, Any]] = None,
+        io_config: Union[Dict[str, Any], "IoConfig", str, Callable] = None,
     ):
         self.framework = framework
         self.model_file_format = model_file_format
         self.composite_parent = None
         self.model_attributes = model_attributes
-        self.io_config = None
+        self._io_config = (
+            validate_config(io_config, IoConfig).dict(exclude_none=True)
```

Comment:
This test test_pytorch_model:test_func_io_config will fail.

Reply:
actually, the test_func_io_config could pass in my local machine